source/_integrations/knx.markdown

```diff
@@ -152,8 +152,138 @@ The user password can be obtained almost the same way as seen in the below scree
+- `trigger.dpt_main` Destination group address main datapoint type number
+- `trigger.dpt_sub` Destination group address sub datapoint type number
+- `trigger.dpt_name` DPT value type name - see Sensor value types
+- `trigger.payload` Raw telegram payload. DPT 1, 2 and 3 yield integers 0..255; other DPT yield lists of integers 0..255
+- `telegram.source` Source individual address
+- `telegram.source_name` Source name
+- `telegram.telegramtype` APCI type of telegram
+- `telegram.timestamp` Timestamp
+- `telegram.unit` Unit according to group address DPT
+- `telegram.value` Decoded telegram payload according to DPT
+
+| Template variable          | Type                        | Project data required |
+|----------------------------|-----------------------------|-----------------------|
+| `trigger.destination`      | string                      | no                    |
+| `trigger.destination_name` | string                      | yes                   |
+| `trigger.direction`        | string                      | no                    |
+| `trigger.dpt_main`         | integer                     | yes                   |
+| `trigger.dpt_sub`          | integer                     | yes                   |
+| `trigger.dpt_name`         | string                      | yes                   |
+| `trigger.payload`          | integer or list of integers | no                    |
+| `telegram.source`          | string                      | no                    |
+| `telegram.source_name`     | string                      | yes                   |
+| `telegram.telegramtype`    | string                      | no                    |
+| `telegram.timestamp`       | timestamp                   | no                    |
+| `telegram.unit`            | string                      | yes                   |
+| `telegram.value`           | any                         | yes                   |
+
+For values that require project data, if the information was not found, or no project file was provided, data will be set to `null`.
+
+#### Examples
+
+Example automation configuration
+
+```yaml
+- alias: Single group address trigger
+  description: ''
+  trigger:
+  - platform: knx.telegram
+    destination: 1/2/3
+    group_value_read: false
+    outgoing: false
+  condition: []
+  action: []
+  mode: single
+```
+
+Example trigger data
+
+```yaml
+variables:
+  trigger:
+    id: "0"
+    idx: "0"
+    alias: null
+    destination: 1/2/3
+    destination_name: Light office brightness
+    direction: Incoming
+    dpt_main: 5
+    dpt_sub: 1
+    dpt_name: percent
+    payload:
+      - 255
+    source: 1.0.51
+    source_name: Dimming actuator 1
+    telegramtype: GroupValueWrite
+    timestamp: "2024-01-09T10:38:28.447487+01:00"
+    unit: "%"
+    value: 100
+context: null
+```
+
 ## Events
 
+<div class='note'>
+
+For automation triggers it is recommended to use the [knx.telegram](#telegram-trigger) trigger instead of `knx_event`.
```

Comment:
```suggestion
For automation triggers it is recommended to use the [knx.telegram](#telegram-trigger) trigger instead of `knx_event`.
```
```suggestion
For automation triggers, it is recommended to use the [knx.telegram](#telegram-trigger) trigger instead of `knx_event`.
```